tests/functional/ingestion/MongoQueueProcessor.js

```diff
@@ -369,11 +369,36 @@ describe('MongoQueueProcessor', function mqp() {
         });
 
         it('should save to mongo a new object key with data', done => {
+            const objKey = `new-${KEY}`;
             const versionKey = `new-${KEY}${VID_SEP}${NEW_VERSION_ID}`;
+            const contentLength = 110;
+            const objLocations = [{
+                key: objKey,
+                start: 0,
+                size: 50,
+                dataStoreName: LOCATION,
+                dataStoreETag: '1-7cccfcef3abdfaba48b2d193cb146074',
```

Comment:
I don't remember why we have `1-` here and not `1:` but maybe it's normal.

Reply:
Mistake but it is ignored in the code.